cmd/kpod/main.go

```diff
@@ -14,7 +14,29 @@ func main() {
 	app.Version = "0.0.1"
 
 	app.Commands = []cli.Command{
+		imagesCommand,
 		launchCommand,
+		rmiCommand,
+	}
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "root",
+			Usage: "path to the root directory in which data, including images,  is stored",
+			Value: "/var/lib/containers/storage",
```

Comment:
These probably should not be constants, I think you can get these values from container storage.